Makefile

```diff
@@ -83,7 +83,7 @@ $(PLATFORMS):
 
 .PHONY: lint
 lint: codegen tools
-	$(GO_LINT) run -v ./...
+	$(GO_LINT) run -v --timeout 5m ./...
```

Comment:
Just curious, under what kind of cases will the linter cost this much time?

Reply:
under GitHub action, the linter will use more than 1m which is the default value of golang-ci lint.